atlas/commands/bootstrap/bootstrap.go

```diff
@@ -49,6 +51,8 @@ var (
 	initializeProfiler = initialize.Bool(flagWithProfiler, false, "initialize the application with a profiling service")
 	initializeHelm     = initialize.Bool(flagWithHelm, false, "initialize the application with the helm charts")
 	initializeExpand   = initialize.String(flagExpandName, "", "the name of the input file for the `expand` command (optional)")
+	initializeKind     = initialize.Bool(flagWithKind, false, "initialize the application with KinD support (optional, only applicable when helm charts are enabled)")
```

Comment:
The atlas-cli seems to have a lot of helpful things turned off like `helm` and `kind`. Perhaps we should make these default==true?

Reply:
We have a lot of disabled features here, and it's not only the last-added things... You're right, we have a lot of extra functionality here, but probably it makes sense these features were enabled in accordance with user requirements, by this user, during the service creation. I assume some devs don't need helmcharts, kind or something else, especially if they are going to use their services as a simple/ standalone apps or just a sandboxes.
BTW, just an idea: in theory, we can create config file and store it somewhere in the system, but I'm not sure regarding the usability. For example, that file will be filled first time when the user creates his first service using atlas-cli, command-line args will be used as these "preferred values". After that, we can use that config as default if no args provided, but command-line args can override these values.

Reply:
Config file is a good idea for future work 🎫